### DISCUSSION TITLE
Add missing tag icons

### DISCUSSION THREAD

**Discussion by pascalwittler** [Traewelling/traewelling#2689] [Ideas / Feature Requests]
The tags "Wagen", "Wagengattung" and "Fahrzeugnummer" do not have an icon which results in

* less space left for the text input value when creating/editing the tags (compare [1] and [2])
* a less beautiful and inconsistent appearance in the list (see [3])

[1] Large enough text input thanks to the icon on the left

![1](https://github.com/Traewelling/traewelling/assets/20669380/2c9398dc-88f1-41f8-9b35-9e9bcd22928b)

[2] Too small text input due to a word as icon fallback

![2](https://github.com/Traewelling/traewelling/assets/20669380/87e0b9bc-2e88-4e91-adea-fb474ab1bbec)

[3] Inconsistent appearance of tags with and without icons

![3](https://github.com/Traewelling/traewelling/assets/20669380/bd3f001c-6ce4-42b1-a506-a1fb2ad61a48)

**Comment by MrKrisKrisu** [comment]
Do you have suggestions for Icons? We are using font awesome free.

**Comment by pascalwittler** [comment]
"Wagen":
https://fontawesome.com/icons/arrow-up-1-9
or
https://fontawesome.com/icons/arrow-right-arrow-left

"Wagengattung":
https://fontawesome.com/icons/list-check

"Fahrzeugnummer":
https://fontawesome.com/icons/tag

I know it's not very satisfying, but these icons may be better than nothing if you are limited to the FontAwesome Free icon pack.

**Comment by MrKrisKrisu** [comment]
Icons added in #4440
